### PR TITLE
Adds Tests for Slider Web Component v3

### PR DESCRIPTION
**Pull request by procload**
## New Behavior

Adds Tests for Slider Web Component v3

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b03b40647507909a151f966be6cd67b65cfcf3f9'>b03b40647507909a151f966be6cd67b65cfcf3f9</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/web-components/src/slider/slider.spec.ts [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+  });
+
+  test('should set a default `max` property of 10 when `max` is not provided', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+            <fluent-slider></fluent-slider>
+        `;
+    });
+
+    await expect(element).toHaveAttribute('max', '10');
```
Comment (halerankin) [inline packages/web-components/src/slider/slider.spec.ts:67]: Are you looking for attribute or JS Property?

**Review by halerankin — APPROVED** [review]

File: packages/web-components/src/slider/slider.spec.ts (hunk 1 above)
Comment (procload) [reply packages/web-components/src/slider/slider.spec.ts:67]: Good catch, will fix!

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**